azuredeploy.json

```diff
@@ -35,14 +35,15 @@
       }
     },
     "vmSize": {
-      "defaultValue": "Standard_DS3",
+      "defaultValue": "Standard_DS2_v2",
```

Comment:
DS2_v2 has 2 cores and 7GB RAM, whereas the [hardware requirements](https://docs.chef.io/install_server_ha.html#hardware-requirements
) are:

Frontend Requirements: 4 cores (physical or virtual), 4GB RAM, 20 GB of free disk space (SSD if on premises, Premium Storage in Microsoft Azure, EBS-Optimized GP2 in AWS)
Backend Requirements: 2 cores (physical or virtual), 8GB RAM, 50 GB/backend server (SSD if on premises, Premium Storage in Microsoft Azure, EBS-Optimized GP2 in AWS)

If this is being deployed in a customer account I don't see why we would make this recommendation.  If anything we need to review the later, more storage-optimized sizes.  We should also not restrict the allowedValues so that customers can customize to their needs (e.g. support F, B series sizes)

Reply:
Thanks @stuartpreston.  The original intent of this fix was to reduce Azure running costs on this quickstart, as discussed with @davymcaleer and @russellseymour.  However, in response to your discussion I've:
* created a task [to upgrade chef-automate-ha to meet the hardware requirements](https://github.com/chef-partners/chef-automate-ha/projects/2#card-14241760).
* raised an [issue about production requirements in general](https://github.com/chef-partners/chef-automate-ha/issues/17) and created a task [to generate further actions against chef-automate-ha](https://github.com/chef-partners/chef-automate-ha/projects/2#card-13883122), i.e., what other aspects of the cluster must be changed to ensure it is a production ready cluster?  If I kill a couple of the backend or frontend servers, does it continue to run without issue?

FYI @trevorghess.